app/gui2/src/components/DocumentationPanel/DocsTags.vue

```diff
@@ -1,64 +1,132 @@
+const skipTags: Doc.Tag[] = ['Icon', 'TextOnly']
 const tags = computed<Doc.Section.Tag[]>(() => {
   return props.tags.flatMap((tag) => {
     if (tag.tag === 'Alias') {
+      // Split Alias tags into separate ones.
       return tag.body.split(/,\s*/).map((body) => ({ tag: 'Alias', body }))
+    } else if (skipTags.includes(tag.tag)) {
+      // Skip certain tags.
+      return []
     } else {
       return tag
     }
   })
 })
 
-interface View {
+// === Sorting ===
+
+const weight: Partial<Record<Doc.Tag, number>> = {
+  Group: 1000,
+  Unstable: 502,
+  Advanced: 501,
+  Deprecated: 500,
+  Alias: 100,
+}
+
+const sortedTags = computed<Doc.Section.Tag[]>(() => {
+  return [...tags.value].sort((a, b) => (weight[b.tag] ?? 0) - (weight[a.tag] ?? 0))
+})
+
+// === View ===
+
+export interface View {
   label: string
   class?: string
   style?: { [key: string]: string | number | undefined }
 }
 
 const defaultView = (tag: Doc.Tag, body: Doc.HtmlString): View => ({
-  label: tag + (body.length > 0 ? `=${body}` : ''),
+  label: (tag + (body.length > 0 ? `: ${body}` : '')).toLowerCase(),
 })
 const tagsMap: Partial<Record<Doc.Tag, (body: Doc.HtmlString) => View>> = {
+  Advanced: (_body) => ({
+    label: 'advanced',
+    style: {
+      'background-color': '#e89d51',
+      color: '#fff',
+    },
+  }),
   Alias: (body) => ({ label: body }),
+  Deprecated: (_body) => ({
+    label: 'deprecated',
+    style: {
+      'background-color': '#e89d51',
+      color: '#fff',
+    },
+  }),
+  Group: (body) => ({
+    label: body.toLowerCase().replace('standard.base.', ''),
+    style: {
+      'background-color': props.groupColor,
+      color: '#fff',
+    },
+  }),
+  Unstable: (_body) => ({
+    label: 'unstable',
+    style: {
+      'background-color': '#e85252',
+      color: '#fff',
+    },
+  }),
 }
 const view = (tag: Doc.Section.Tag): View => {
   const maybeView = tagsMap[tag.tag]
   return maybeView ? maybeView(tag.body) : defaultView(tag.tag, tag.body)
 }
+const views = computed<View[]>(() => sortedTags.value.map(view))
+
+// === Show/hide excess tags. ===
+
+const containerRef = ref<HTMLDivElement | null>(null)
+const parentOffset = computed(() => containerRef.value?.offsetTop ?? 0)
+
+const hiddenTags = ref(0)
+const tagWasHidden = () => (hiddenTags.value += 1)
+const someTagsAreHidden = computed(() => hiddenTags.value > 0)
+
+const showAllTags = ref(false)
+const showEllipsis = computed(() => someTagsAreHidden.value && !showAllTags.value)
+watch(
+  () => props.tags,
+  () => {
+    showAllTags.value = false
+    hiddenTags.value = 0
+  },
+)
 </script>
 
 <template>
-  <div v-if="tags.length > 0" class="Tags">
-    <div
-      v-for="(tag, index) in tags"
-      :key="index"
-      :class="view(tag).class || 'tag'"
-      :style="view(tag).style || {}"
-    >
-      {{ view(tag).label }}
-    </div>
+  <div v-if="views.length > 0" ref="containerRef" class="Tags">
+    <template v-for="(v, _i) in views" :key="_i">
```

Comment:
`_i`?

Reply:
LS reports it as unused, perhaps my editor’s issue